score/cronjobs/humanitec.score.yaml

```diff
@@ -1,12 +1,47 @@
 apiVersion: humanitec.org/v1b1
+# Select the "default-cronjob" built-in Workload Profile
 profile: humanitec/default-cronjob
 
 spec:
+  # Define the CronJob schedules
   schedules:
     6-hour-run:
       containers:
         main-container:
           args:
             - "--hours"
             - "6"
-      schedule: "15 1,7,13,19 * * *"
\ No newline at end of file
+      # 🚧 schedule: "15 1,7,13,19 * * *"
+      schedule: "* * * * *"
+  cronjob:
+    # 🚧 There is a bug in the Chart preventing from rendering these items properly.
+    # 🚧 See https://github.com/humanitec/deployment-helm-charts/pull/86 -> merged, still needs to be published
```

Comment:
Releasing in  https://github.com/humanitec/workload-profiles-service/pull/405

Reply:
Thanks @astromechza ! I just re-tested and is seems the service is not yet promoted to production at this time, correct? Double-checking before merging the present PR.